dell/xps/13-9360/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, pkgs, ... }:
+let
+  firmware_qca6174 = pkgs.callPackage ./firmware_qca6174.nix {};
+in
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+  ];
+  boot.kernelModules = ["kvm-intel"]; # should this be in common/cpu/intel?
+  boot = {
+    loader = {
+      systemd-boot.enable = lib.mkDefault true;
+      efi.canTouchEfiVariables = lib.mkDefault true;
+    };
+
+    kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;
```

Comment:
Does it only work with the latest kernel? (Current LTS kernel is 4.14.x)

Reply:
everything higher than 4.11 should work without kernel patches, but not all of them work as well as others. 4.14.x should be fine, but I haven't tested that in forever.
Is https://patchwork.kernel.org/patch/9676073/ in 4.14?